doc/api/cli.md

```diff
@@ -2780,6 +2780,13 @@ equivalent to using the `--redirect-warnings=file` command-line flag.
 added:
  - v13.0.0
  - v12.16.0
+changes:
+  - version:
+     - REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/52905
+    description:
+      Remove the possibility to use this env var with
+      kDisableNodeOptionsEnv for embedders.
```

Comment:
Does this information need to be added to the user-visible text? Or too much of an edge case? I'm guessing too much of an edge case to be worth mentioning, but asking just in case I'm wrong.

Reply:
Eh, never mind. This seems like a bug fix. The current behavior seems (to me, at least) surprising. (But I'll defer to others more knowledgable about how this stuff gets used if they disagree with me on that.)